src/Orc.LicenseManager.Server/Services/AccountService.cs

```diff
@@ -22,6 +22,8 @@ public AccountService()
         {
         }
 
+#pragma warning disable IDISP001 // Dispose created
```

Comment:
I think this is the best solution here as we don't won't to dispose or somehow decide the lifecycle of DbContext

Reply:
And this is the server part we don't really care much about anyway, so all is good.